cfme/tests/intelligence/test_download_report.py

```diff
@@ -43,24 +41,13 @@ def report():
     return reports.CannedSavedReport(path, reports.queue_canned_report(*path))
 
 
-@pytest.skip  # To be removed when we have solved the Docker issue
```

Comment:
@psav Do we have the files download solved, since the browser runs in a separate container?